src/BitbucketContext/Domain/Entity/Analysis.php

```diff
@@ -32,7 +32,10 @@ class Analysis
     /** @var ArrayCollection<Violation> */
     private $violations = [];
```

Comment:
Property Regis\BitbucketContext\Domain\Entity\Analysis::$violations (Doctrine\Common\Collections\ArrayCollection) does not accept default value of type mixed[].